lib/HTML/Renderers/LinkRenderer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\HTML\Renderers;
+
+use Doctrine\RST\Environment;
+
+final class LinkRenderer extends \Doctrine\RST\Renderers\LinkRenderer
```

Comment:
I'm surprised phpcs is OK with this :thinking:

Reply:
php cs was not ok with naming the latter LinkRendererInterface...

Reply:
Oh right. How about aliasing it to `LinkRendererInterface` at import time?